server/src/main/java/com/genymobile/scrcpy/ControlMessage.java

```diff
@@ -11,11 +11,12 @@
     public static final int TYPE_INJECT_SCROLL_EVENT = 3;
     public static final int TYPE_BACK_OR_SCREEN_ON = 4;
     public static final int TYPE_EXPAND_NOTIFICATION_PANEL = 5;
-    public static final int TYPE_COLLAPSE_NOTIFICATION_PANEL = 6;
-    public static final int TYPE_GET_CLIPBOARD = 7;
-    public static final int TYPE_SET_CLIPBOARD = 8;
-    public static final int TYPE_SET_SCREEN_POWER_MODE = 9;
-    public static final int TYPE_ROTATE_DEVICE = 10;
+    public static final int TYPE_EXPAND_SETTINGS_PANEL = 6;
+    public static final int TYPE_COLLAPSE_PANELS = 7;
+    public static final int TYPE_GET_CLIPBOARD = 8;
+    public static final int TYPE_SET_CLIPBOARD = 9;
+    public static final int TYPE_SET_SCREEN_POWER_MODE = 10;
+    public static final int TYPE_ROTATE_DEVICE = 11;
```

Comment:
Not fully sure about this (and related) as it will make the previous server incompatible.

1. On one hand, as I did,
    1. this breaks backwards compatibility.
    2. making this change allows cleaner code on both ends.
2. On the other hand, by just adding at the end
     1. Keeps backwards compatibility
     2. The code is less readable and maintainable.
 
I ended up going with `1`, but I don't mind changing to `2` if you prefer.
Maybe doing as `2` and change to `1` soon.

Reply:
> Not fully sure about changing the value of the constants as I did (and related) as it will make the previous server incompatible.

That's totally acceptable, servers are not expected to be compatible at all with any other scrcpy client version. This would add annoying constraints for absolutely no benefit for scrcpy.